topics/openapi/index.md

```diff
@@ -0,0 +1,23 @@
+---
+topic: openapi
+aliases: open-api, swagger-spec, swagger-specification, openapi-spec, openapi-specification, open-api-specification, oas, openapi-description
+display_name: OpenAPI Specification
+related: json-schema, raml, rest-api
+created_by: OpenAPI Initiative, SmartBear Software, Tony Tam, Darrel Miller, Jeremy Whitlock, Marsh Gardiner, Mike Ralphson, Ron Ratovsky, Uri Sarid, Jason Harmon
+released: August 10, 2011
+url: https://www.openapis.org/
+wikipedia_url: https://en.wikipedia.org/wiki/OpenAPI_Specification
+github_url: https://github.com/OAI/OpenAPI-Specification
+---
+The **OpenAPI Specification** (OAS), previously known as the **Swagger Specification**, is a specification for a programming language-agnostic, machine-readable interface definition language for describing, producing, consuming and visualizing web services. OpenAPI documents describe API services and are represented in YAML or JSON formats.
+
+# Advantages of Using OpenAPI
+* **Description Validation and Linting**: Check that your description file is syntactically correct and adheres to a specific version of the Specification and the rest of your team’s formatting guidelines.
+* **Data Validation**: Check that the data flowing through your API (in both directions) is correct, during development and once deployed.
+* **Documentation Generation**: Create traditional human-readable documentation based on the machine-readable description, which always stays up-to-date.
+* **Code Generation**: Create both server and client code in any programming language, freeing developers from having to perform data validation or write SDK glue code, for example.
+* **Graphical Editors**: Allow easy creation of description files using a GUI instead of typing them by hand.
+* **Mock Servers**: Create fake servers providing example responses which you and your customers can start testing with before you write a single line of code.
+* **Security Analysis**: Discover possible vulnerabilities at the API design stage instead of much, much later.
+* A non-proprietary format
```

Comment:
Oh, well, I didn't intend it to sound like that and, sure, there's always the URL. I actually thought of it more as a list of features, that might be represented by another GitHub topic, so it could potentially be linked

Reply:
No worries at all. I was going to suggest replacing this with links to the topics that are related, but then that would conflict with the actual related topics list 🤔, but I think removing it is probably the right call for now.

Reply:
Yeah, sometimes, we have topics for this, sometimes, we don't. And sometimes, there are more than are allowed so it would make sense to list them separately. Or sometimes, they are to vague or to narrow.

But I'll remove them in this particular case. Might be the best for now.